### DISCUSSION TITLE
大佬，PC端能否加个代理设置

### DISCUSSION THREAD

**Discussion by cyberkingkong** [xiaohucode/yidaRule#90] [Ideas]
大佬，PC端能否加个代理设置，这样就不用开全局代理了。感谢大佬

**Comment by xiaohucode** [comment]
1.1.14版本已添加代理

**Comment by frank77177** [comment]
这个代理设置要怎么弄？

**Reply by zentst** [reply]
以clash为例，clash代理端口是7897，那代理设置里就填写 http://127.0,0,1:7897，保存。
我这样配置就能导入网络规则了（github要翻墙）但问题是有些站点还是打不开，

**Reply by frank77177** [reply]
小火箭节点共享代理。我填了也不行
